src/codegen_call_json.cpp

```
@@ -14,202 +14,197 @@ bool CodeGen::isJsonValue(llvm::Value *val) {
-        llvm::Value *isErr = builder_.CreateICmpNE(status,
-            llvm::ConstantInt::get(i64Ty_, 0), "json_bool_err");
-        llvm::StructType *resTy = getResultType(i1Ty_, errorTy_);
-        return emitResultBranch(isErr, resTy,
-            [&]() {
-                llvm::Value *loaded = builder_.CreateLoad(i64Ty_, outSlot, "json_bool_i64");
-                llvm::Value *boolVal = builder_.CreateTrunc(loaded, i1Ty_, "json_bool_val");
-                return buildOkValue(boolVal, resTy);
-            },
-            [&]() { return buildErrValue(buildErrorFromRuntime(), resTy); });
-    }
+llvm::Value *CodeGen::emitJsonToStr(const CallExpr &e) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)